test/support/contracts/multicall3.sol

```diff
@@ -0,0 +1,300 @@
+// SPDX-License-Identifier: MIT
```

Comment:
If we are using the signed transaction from Multicall then why do we need the actual implementation here?